dataverse-persistence/src/main/resources/db/migration/V49__alterAuthorAffiliationDisplayFormat.sql

```diff
@@ -0,0 +1 @@
+UPDATE datasetfieldtype SET displayformat='- #VALUE' WHERE name='authorAffiliation';
```

Comment:
This change will have inpact on code in `AbstractGlobalIdServiceBean`
```
                if (author.getAffiliation() != null && !author.getAffiliation().getDisplayValue().isEmpty()) {
                    creatorsElement.append("<affiliation>" + author.getAffiliation().getDisplayValue() + "</affiliation>");
                }
```
I'm pretty sure we want to `author.getAffilitation().getValue()` but we could check that.

Reply:
Hmm, yeah Im not sure why the display value is used in this place since it is xml information for datacite.